source/config/configSpec.py

```diff
@@ -249,6 +249,10 @@
 
 [development]
 	enableScratchpadDir = boolean(default=false)
+
+[featureFlag]

```

Comment:
If we're creating a new config spec section for feature flags, do we want to move the existing ones there? For consistency maybe this should be part of speech settings in the config (but advanced in the GUI)?

Reply:
In the case of this PR there is no intention of this becoming a relevant config option for most users. There is no plan to move this option into the speech section. After wider testing the default will change, and after some time has passed the GUI control can be removed assuming there are no problems. For stable GUI controls I think there is value in the config spec section matching the category. Feature flags are a little different, it helps us to introduce a change in behavior in a controlled way, allowing users to easily revert to prior behavior. I don't think there is much tangible benefit to moving preexisting feature flags to this section.